Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python
```

Comment:
It's good to use the exact image tag otherwise it will be the latest that could lead to unexpected behavior